client/network/src/config.rs

```diff
@@ -181,8 +180,6 @@ impl TryFrom<String> for MultiaddrWithPeerId {
 pub enum ParseErr {
 	/// Error while parsing the multiaddress.
 	MultiaddrParse(multiaddr::Error),
-	/// Multihash of the peer ID is invalid.
-	InvalidPeerId,
```

Comment:
Great to see this error case removed from runtime, but instead made impossible at compile time.